[FW][IMP] ecommerce: update product/attribute page (#13037)
odoo/documentation: content/applications/websites/ecommerce/products.rst

.. _ecommerce/products/product-page:

> Product page customization

Reviewer: Comment about the subsections *Backend* and *Frontend*:

Instead of using "how" or "where" you do it (i.e., from the backend or frontend) as a heading, I would try to use the "why" or "what type of customization you do where. For example, in this case, the backend is more used to change the *product information* itself (product name, variants, documents, and description), while the frontend is used to actually customize the page, like we can customize other website pages, if that makes sense. So I'd use: *Product information* and *Product page* (not super sure about this second one), maybe? What do you think?

I would also change the first sentence of each section accordingly.

==========================

.. _ecommerce/products/customization:

Backend
-------
.. _ecommerce/products/product-form:

To customize the **product page** from the backend, navigate to :menuselection:`Website -->
eCommerce --> Products` and select the product. You can configure the product page from the form
by adding :ref:`variants <ecommerce/products/product-variants>`, :ref:`digital documents
<ecommerce/products/digital-files>`, or :ref:`translating <ecommerce/products/translation>` content.

> You can also add an additional ecommerce description for marketing purposes that will only
> appear on the ecommerce product page under the product name. To do so, go to the :guilabel:`Sales`

Reviewer: ```suggestion
You can also add a ecommerce-specific product description to be displayed below the product name on the product's page. To do so, go to the :guilabel:`Sales`
```
Why I suggest this change:

- Since users can use this field for whatever they want technically, I suggest removing *for marketing purposes* and simply describing where the text is displayed.
- We try to avoid unnecessary future tense.

tab, scroll down to the :guilabel:`Ecommerce description` section, and add a description. Use Odoo's

> :doc:`rich-text editor <../../essentials/html_editor>` features to personalize your content. You

Reviewer: great idea to mention this and link to the relevant doc! :slightly_smiling_face: 

I would just say "to personalize the content" instead of "your content" as we try to avoid using you/your when it's not essential.
